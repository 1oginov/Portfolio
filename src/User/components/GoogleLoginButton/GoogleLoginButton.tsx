import * as React from 'react';

import ProviderLoginButton from '../ProviderLoginButton';

const GoogleLoginButton = () => (
  <ProviderLoginButton provider="google">
    Sign in with Google
  </ProviderLoginButton>
);

export default GoogleLoginButton;
