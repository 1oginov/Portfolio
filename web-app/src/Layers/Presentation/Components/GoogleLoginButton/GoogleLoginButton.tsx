import * as React from 'react';

import { ProviderLoginButton } from 'Layers/Presentation/Components/ProviderLoginButton';

export const GoogleLoginButton: React.FunctionComponent = () => (
  <ProviderLoginButton provider="google">
    Sign in with Google
  </ProviderLoginButton>
);
