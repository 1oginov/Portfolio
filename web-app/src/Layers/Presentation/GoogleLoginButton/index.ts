import { GoogleLoginButton as GoogleLoginButtonComponent } from './GoogleLoginButton';

export const GoogleLoginButton = GoogleLoginButtonComponent;
