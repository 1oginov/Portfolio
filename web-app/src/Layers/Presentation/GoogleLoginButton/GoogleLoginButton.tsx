import * as React from 'react';

import { ProviderLoginButton } from 'Layers/Presentation/ProviderLoginButton';

export const GoogleLoginButton: React.FunctionComponent = () => (
  <ProviderLoginButton provider="google">
    Sign in with Google
  </ProviderLoginButton>
);
