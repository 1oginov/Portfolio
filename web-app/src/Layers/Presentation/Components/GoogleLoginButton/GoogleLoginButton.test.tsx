/* eslint-disable import/no-extraneous-dependencies */

import { shallow } from 'enzyme';
import * as React from 'react';

import { GoogleLoginButton } from './GoogleLoginButton';

it('matches snapshot', () => {
  const wrapper = shallow(<GoogleLoginButton />);

  expect(wrapper).toMatchSnapshot();
});
