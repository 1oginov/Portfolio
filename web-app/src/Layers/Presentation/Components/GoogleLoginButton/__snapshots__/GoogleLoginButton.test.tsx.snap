// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Connect(withHandlers(ProviderLoginButton))
  provider="google"
>
  Sign in with Google
</Connect(withHandlers(ProviderLoginButton))>
`;
